overlays/nodejs/serverless/function.ts

```diff
@@ -111,14 +121,16 @@ export class Function extends pulumi.ComponentResource {
             entryPoint: handlerName,
             sourceArchiveBucket: this.bucket.name,
             sourceArchiveObject: this.bucketObject.name,
-            triggerHttp: true,
+            triggerHttp: !(options.triggerBucket || options.triggerTopic),
```

Comment:
These values are `Input<T>` so you would likely need to do `pulumi.all([options.triggerBucket, options.triggerTopic]).apply(([triggerBucket, triggerTopic]) => !(options.triggerBucket || options.triggerTopic))`.   I suspect as written this will not work correctly.

Reply:
But actually also not sure this is what we want.  Should we just expose `triggerHttp` as a property like in the TF provider and pass it through here.  That would be a breaking change (it would not be `true` by default) - but I think it's more likely the right behaviour here to align with the underlying provider.